src/libostree/libostree-devel.sym

```diff
@@ -20,12 +20,17 @@
    - uncomment the include in Makefile-libostree.am
 */
 
+LIBOSTREE_2022.2 {
+global:
+  ostree_sysroot_auto_cleanup;
```

Comment:
New public API feels overkill for this if the expectation is that it will be invoked by a separate systemd unit.  Related below:

Reply:
I guess this was with an eye toward having the systemd unit disabled and letting the management daemon handle it itself. But I did do this in endlessm/eos-updater#298 without needing the API. I do think it's cleaner to have the semantics contained within libostree, though.